_pages/logs/rclog/index.md

```diff
@@ -40,6 +40,53 @@ header:
 
 ## 2019
 
+### 2019-12-11
+#### 4.19.89, 5.3.16, 5.4.3
+<!-- sla <24h 3 -->
```

Comment:
Can you fix this to not have an "h" after <24? Try running ./get_sla.py to see why.

Reply:
Done.